lib/ipmi_fru.c

```diff
@@ -1520,7 +1520,7 @@ static void fru_area_print_multirec(struct ipmi_intf *intf,
  * returns : TRUE if data changed
  * returns : FALSE if data not changed
  */
-int ipmi_fru_query_new_value(uint8_t *data, int offset, size_t len)
+static int ipmi_fru_query_new_value(uint8_t *data, int offset, size_t len)
```

Comment:
`static` must be on a separate line. as well as `int`.
https://github.com/ipmitool/ipmitool/wiki/Coding-Standards#function-definition
That's been inherited from the old coding style used at SourceForge by this project.
As always, it hasn't been strictly followed, as you can see. But let's follow it at least where we change things.